android/src/main/AndroidManifestNew.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+</manifest>
```

Comment:
Do we need this AndroidManifestNew along with AndroidManifest?

Reply:
Yes,
<img width="633" alt="Screenshot 2024-02-13 at 09 53 18" src="https://github.com/Judopay/JudoKit-ReactNative/assets/85407970/7c3bebd8-f46e-4c52-9c64-f977e534a0cf">